### PR TITLE
[WIP] Adds nodejs-axios codegen

### LINKED ISSUE #135: Can you please add the support for axios

### PR DESCRIPTION
**Pull request by someshkoli**
Closes #135
Adds support for axios codegen as requested in #135
snippet samples:
post req:
image
get req:
image

### CODE REVIEW

**Comment by someshkoli** [comment]
hey @shreys7 , the build is failing due to timeout in java-okhttp can you check once if there's something missing  ?

**Comment by emanuelet** [comment]
possible duplicate, https://github.com/postmanlabs/postman-code-generators/pull/232

**Comment by someshkoli** [comment]
Not exactly,  we both made pr almost at the same time without knowing each other's work. There's a closed PR #236. Had to close this as the branch was bit messed up.

File: codegens/nodejs-axios/test/newman/newman.test.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      },
+      testConfig = {
+        // filename along with the appropriate version of the file. This file will be used to run the snippet.
+        fileName: 'codesnippet.js',
+        // Run script required to run the generated code snippet
+        runScript: 'node codesnippet.js',
+        // Compile script required to compile the code snippet
+        compileScript: '',
+        // Array of name of collections for which newman tests has to be skipped.
+        skipCollections: ['sameNameHeadersCollection', 'formdataCollection', 'formdataFileCollection']
```
Comment (shreys7) [inline codegens/nodejs-axios/test/newman/newman.test.js:19]: Why are these collections skipped?

**Comment by shreys7** [comment]
I think URL, method, body, etc. can also be provided in the config object of Axios. Can you check it once?

**Comment by someshkoli** [comment]
Yeah it can be done. Axios can be used in 2 ways and this is one of them and ig majority of people use this approach.

File: codegens/nodejs-axios/test/newman/newman.test.js (hunk 1 above)
Comment (someshkoli) [reply codegens/nodejs-axios/test/newman/newman.test.js:19]: i don't actually remember :\. ill test again maybe

**Comment by shreys7** [comment]
We will look at it and take it up if the users are requesting the other approach. For now, we would like to have method, URL, body, etc. to be part of the config object @someshkoli

**Comment by shreys7** [comment]
any updates @someshkoli ?

**Comment by someshkoli** [comment]
Nah the tests were still failing. Was waiting for your pr to get merged #246.

**Comment by shreys7** [comment]
I am asking about the configuration part. The body, method etc.

**Comment by someshkoli** [comment]
Yeah it is

**Comment by someshkoli** [comment]
@shreys7  I'm stuck at one request. And since there is already a perfect implementation ready #232 . Can you please merge his version.

**Comment by shreys7** [comment]
Okay. As you have good enough context too, can you go through #232 and add your review there? @someshkoli

**Comment by someshkoli** [comment]
Sure I'll do that.

**Comment by shreys7** [comment]
Closing this, focussing on #232

**Pull request closed without merging**